homeassistant/components/volvo/strings.json

```diff
@@ -9,6 +9,7 @@
         "title": "[%key:common::config_flow::title::reauth%]"
       },
       "api_key": {
+        "description": "Get your API key from the Volvo developer portal (https://developer.volvocars.com/account/#your-api-applications).",
```

Comment:
URLs should be put in via description placeholders. You can also use markdown here